README.md

```diff
@@ -53,12 +53,35 @@ The action can be configured with the following arguments:
   equivalent of what would be passed to the `pulumi login` command. The action
   will login to the appropriate backend on your behalf provided it is configured
   with the correct access credentials for that backend.
+
 - `comment-on-pr` - (optional) If `true`, then the action will add the results
   of the Pulumi action to the PR
 
 - `github-token` - (required if comment-on-pr) A GitHub token that has access
   levels to allow the Action to comment on a PR.
 
+### Extra options
```

Comment:
We should add a small comment here to suggest that these extra options, are the equivalent of the command line args